helm/charts/oathkeeper/templates/configmap-rules.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "oathkeeper.fullname" . }}-rules
```

Comment:
This isn't the only configmap, any particular reason why the `-rules` was removed?